packages/editor/src/store/reducer.js

```diff
@@ -402,4 +403,5 @@ export default combineReducers( {
 	listViewPanel,
 	listViewToggleRef,
 	publishSidebarActive,
+	dataviews: dataviewsReducer,
```

Comment:
This is named like that because the plan is to have a separate package eventually? Because the actions are used beyond the Data Views, so I don't really get why name it like that.

Reply:
yes, I can rename, what would you suggest? I guess you can think of it as the package name.